opencog/cogserver/server/AgentRunnerBase.cc

```diff
@@ -146,7 +146,7 @@ void AgentRunnerBase::run_agent(AgentPtr a)
             a->classinfo().id.c_str(), elapsed_secs, mem_used, atoms_used,
             cycle_count);
 
-    cogserver().systemActivityTable().logActivity(a, elapsed, mem_used,
+    sat->logActivity(a, elapsed, mem_used,
```

Comment:
@ferrouswheel this line seems to be causing segfault...atleast in the HebbianCreationModuleUTest.

Reply:
I get this error both before and after my commit.
```
2/19 Test  #2: HebbianCreationModuleUTest .......***Failed    0.36 sec
Running cxxtest tests (1 test)
In HebbianCreationModuleUTest::testHebbianLinkCreation:
/home/joel/work/opencog/opencog/tests/attention/HebbianCreationModuleUTest.cxxtest:77: Error: Test failed: Parameter MAX_AF_SIZE has no associated value. (/home/joel/work/opencog/opencog/opencog/attention/AttentionParamQuery.cc:76)
Failed 1 and Skipped 0 of 1 test
Success rate: 0%
;;; Stat of /usr/local/share/opencog/scm/opencog/attention/default-param-values.scm failed:
;;; In procedure stat: No such file or directory: "/usr/local/share/opencog/scm/opencog/attention/default-param-values.scm"
;;; Stat of /usr/local/share/opencog/scm/opencog/attention/default-param-values.scm failed:
;;; In procedure stat: No such file or directory: "/usr/local/share/opencog/scm/opencog/attention/default-param-values.scm"
```

Maybe that's why I didn't notice a segfault, the test was already failing for me?

Reply:
That one is a different one. the param file is being read from installation directory. Installing opencog will resolve it.